MekHQ/src/mekhq/campaign/Campaign.java

```diff
@@ -6231,6 +6231,17 @@ public int getUnitRatingMod() {
                         : rating.getModifier();
     }
 
+    /**
+     * This is a better method for pairing AtB with IOpts with regards to Prisoner Capture
+     */
+    public int getUnitRatingAsInteger() {
+        if (getCampaignOptions().useDragoonRating()) {
```

Comment:
This seems like if you're using the CamOps unit rating, then it'll always return a rating C, which may not be the desired effect.

Reply:
Confusingly named variable... useDragoonsRating enables Unit Rating. I've got an idea for a rework of it... but not here nor now.